app/controllers/search_controller.rb

```diff
@@ -112,15 +104,11 @@ def query
       :restrict_to_archetype
     ].present?
 
-    if rate_limit_errors
-      result =
-        Search::GroupedSearchResults.new(
-          type_filter: search_args[:type_filter],
-          term: params[:term],
-          search_context: context,
-        )
-
-      result.error = I18n.t("rate_limiter.slow_down")
+    if rate_limit_search
+      return(
```

Comment:
```suggestion
      return (
```

Nitpick, surprised the linters don't raise a warning here.

Reply:
It looks like `syntax_tree` is formatting it like this on purpose. Editing it to the suggestion and applying `syntax_tree` brings removes the space.

Reply:
Ok, let's leave it as is.